### PR TITLE
feat(consonant-blocks): more blocks added for the artist hub 2.2 pages

### PR DESCRIPTION
**Pull request by So1ene**
- Added Dropdown menus to Header, for both mobile and desktop, with full accessibility.
- Added breadcrumbs block
- Added sitemap block (above footer)
- Added ability to load embed blocks (example: header, footer)
- Added variants to separator block: 'Small', 'Left', 'Right', 'Accent'
- Added CTA to cards block
- Update widths and responsiveness of different amount of Cards blocks to match XD mockup
- Added 'nocrop' variant for cards block where the image should not be horizontally cropped (example: fonts)
- Added ability to add a caption to Marquee block (example: 'Artist Name')
- External links now open in new tab
- Updated spacing between sections/blocks to match the XD mockup

[(XD Mockup)](https://xd.adobe.com/view/7bed0802-e0e1-4b31-a348-01fd945980db-2ebe/)

URL for testing:

- https://consonant-blocks--pages--webistry-development.hlx3.page/stock/en/artisthub/drafts/artisthub-2.2/

### CODE REVIEW

File: templates/consonant/blocks/header/header.js [hunk 1]
```diff
@@ -9,11 +9,45 @@
...
+  const { href } = window.location;
+
+  if (href.includes('artisthub')) {
+    // Hard-coded for artist-hub pages...
+    let path = '/artisthub';
+    if (href.includes('/drafts/artisthub-2.2')) {
+      path = '/artisthub/drafts/artisthub-2.2';
+    }
+    let page = '';
+    if (href.includes(`${path}/get-started`)) {
```
Comment (fullcolorcoder) [inline templates/consonant/blocks/header/header.js:28]: Switch case might make more sense here

File: templates/consonant/blocks/sitemap/sitemap.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
+main .section-wrapper.sitemap-container {
+    background-color: #F8F8F8;
```
Comment (fullcolorcoder) [inline templates/consonant/blocks/sitemap/sitemap.css:2]: --color-gray-100

File: templates/consonant/styles/styles.css [hunk 3]
```diff
@@ -484,3 +501,104 @@ main .figure .caption {
...
 }
+
+
+/* Default footer */
+footer {
+	font-size: 14px;
+	display: flex;
+	justify-content: center;
+	flex-direction: column;
+  text-align: center;
```
Comment (fullcolorcoder) [inline templates/consonant/styles/styles.css:512]: little spacing :)

**Review by fullcolorcoder — APPROVED** [review]
Nice! Approving with a few small comments/suggestions. Nothing critical. Good lighthouse score in local testing too.

**Review by fullcolorcoder — APPROVED** [review]

**Review by seanchoi-workfront — APPROVED** [review]

**Review by AddumOne — APPROVED** [review]

**Review by seanchoi-workfront — APPROVED** [review]

**Pull request merged**